Tasks/QuickPerfTest/task.json

```diff
@@ -110,6 +119,22 @@
                 "2": "Self-provisioned agents"
             }
         },
+        {
+            "name": "selfProvisionedRig",
+            "type": "string",
+            "label": "Load test rig",
+            "required": false,
+            "defaultValue": "",
```

Comment:
the default value can be "default" and it should be visible only when the option in machinetype is selected as "Self-Provisioned rig"

Reply:
Great! I did not know it was possible, so I just discovered the `visibleRule` option. Tomorrow I will update the PR.

Reply:
`visibleRule` added to both Tasks